### DISCUSSION TITLE
New Game: Enshrouded

### DISCUSSION THREAD

**Discussion by lordimac** [GameServerManagers/LinuxGSM#4463] [Ideas]
Please add Enshrouded as Dedicated Server.
https://enshrouded.zendesk.com/hc/en-us/sections/16050842957085-Multiplayer-and-Server-Hosting

**Comment by lordimac** [comment]
My bad, Windows only (for now). Closed.

**Reply by nuadaxxx** [reply]
It would be great if you could do an install including wine fully automatically

it harms not the latency

**Comment by tonycoculuzzi** [comment]
FWIW: Other server managers have Enshrouded servers working using Proton/Wine

**Reply by h3o66** [reply]
In gerneral is more support , testing and code needed if wine should be supported for linuxgsm.
